LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 TheFarGG & TheGenocides
+Copyright (c) 2022 TheFarGG & TheGenocides
```

Comment:
Better to use 2021-present, people can see the year where the repo was started.

Reply:
kk

Reply:
just LICENSE?

Reply:
> just LICENSE?

All the license messages. One in [LICENSE](https://github.com/PyTweet/PyTweet/blob/master/LICENSE) file and [readme.md](https://github.com/PyTweet/PyTweet/blob/master/README.md)

Reply:
> > just LICENSE?
> 
> All the license messages. One in [LICENSE](https://github.com/PyTweet/PyTweet/blob/master/LICENSE) file and [readme.md](https://github.com/PyTweet/PyTweet/blob/master/README.md)

oghey